GAL22v10_counter8.pld

```diff
@@ -0,0 +1,95 @@
+GAL22V10
+Counter
+
+Clock NC    NC    NC    NC    NC    NC    NC    NC    NC     Reset     GND
+/OE   Q0    Q1    Q2    Q3    Q4    Q5    Q6    Q7    RO     NC     VCC
+
+AR = Reset
```

Comment:
whoa, does this asynchronously reset the counter to 0 whenever `Reset` goes high? That's a 22v10-only feature, I take it?